src/main/java/com/google/devtools/build/lib/skylarkbuildapi/cpp/CcModuleApi.java

```diff
@@ -633,7 +633,7 @@ LibraryToLinkT createLibraryLinkerInput(
       parameters = {
         @Param(
             name = "owner",
-            doc = "List of <code>LibraryToLink</code>.",
+            doc = "The label of the target that produced any files used in this input.",
```

Comment:
I think it should probably be "all" rather than "any"

Reply:
Done.